sessions/project-update-lightning-talks.md

```diff
@@ -0,0 +1,44 @@
+## Session(s): Project Update Lightning Talks 
+
+_Short project updates in rapid succession. No more than 30-45 minutes at a time (6-9 updates). This can/should be done more than once at the meetings to give at least 12-18 projects exposure._ 
+
+**Proposed by** @flyingzumwalt  
+**Led by** @flyingzumwalt   
+**Duration** 30-45 minutes  
+**Who Should Attend:** Everyone  
+**Who has to be there in order for it to work:** People who want to give a project update.
+
+## Overview
+
+- **Objective**: Allow many projects to disseminate information across the entire set of attendees. 
+- **Activity**: In a lecture format, allow a group of people to give pre-prepared 5-minute updates on their projects. 
+- **Outcome**: Recordings of all the project updates and shared context for conference attendees to operate from.
+
+## Details
+
+_This is a running list. Suggest more via PRs, or contact @flyingzumwalt to add something_
+
+Here's a sketch of updates we could have on Day 1 and Day 2. People might want to do another session of updates during the unconference on Day 3.
+
+### Project Updates: Day 1
+- IPFS ecosystem overview
+- js-ipfs 
```

Comment:
@alanshaw let's pair rock this party together for js-ipfs!

Reply:
🙏